### ISSUE TITLE
Add php 7.3 to travis build

### ISSUE THREAD

**Issue by marein** [marein/php-nchan-client#3]

**Comment by marein** [comment]
The build is running for php 7.3. Because `xdebug` is currently not available for php 7.3, the coverage report isn't generated.

```
Error: No code coverage driver is available
```

This causes the `php-coveralls` library to fail.

```
[Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]  
  coverage_clover XML file is not readable: /home/travis/build/marein/php-nch    
  an-client/build/logs/clover.xml
```

**Comment by marein** [comment]
We need to retry/restart the build as soon as `travis` has `xdebug` available for `php 7.3`.

**Comment by marein** [comment]
`xdebug` for `php 7.3` is now available in `travis`. It worked after the restart.
